internal/entities/user.go

```diff
@@ -1 +1,11 @@
 package entities
+
+type User struct {
+	ID         int    `json:"id"`
+	Username   string `json:"username"`
+	Password   string `json:"password"`
+	Email      string `json:"email"`
+	VerifiedAt string `json:"verified_at"`
+	CreatedAt  string `json:"created_at"`
+	UpdatedAt  string `json:"updated_at"`
```

Comment:
Any reason for not doing this?
```suggestion
	CreatedAt  time.Time `json:"created_at"`
	UpdatedAt  time.Time `json:"updated_at"`
```